test/bugs-reported/bug6.chpl

```diff
@@ -0,0 +1,14 @@
+use Queue;
```

Comment:
Ohh..I see. Cool. I'll definitely refer to the release examples for use statement later.